tool_quality_checks/db.py

```diff
@@ -141,6 +139,20 @@ def create_schema(self, schema_name, drop_schema=False):
         )
         self.execute_sql_statement(sql_statement=create_schema_statement, fetch=False)
 
+    def create_preset_view_from_dictionary(
+        self, view_dictionary, view_table, view_schema, drop_view=True
+    ):
+        """
+        Drops and Creates a view from a dictionary
```

Comment:
Docstrings voor functies/methodes moet je idealiter als commando's schrijven:

```suggestion
        Drop and create a view from a dictionary
```